.github/workflows/smoke-checks.yml

```diff
@@ -19,7 +19,7 @@ concurrency:
 
 env:
   HOMEBREW_NO_INSTALL_CLEANUP: 1 # Disable cleanup for homebrew, we don't need it on CI
-  IOS_SIMULATOR_DEVICE: "iPhone 14 Pro (16.4)"
+  IOS_SIMULATOR_DEVICE: "iPhone 15 Pro (17.2)"
```

Comment:
@testableapple We need to re-record all snapshots by using `overrideRecording = true`, so that tests will pass on both CI and Locally (Hopefully)